godotproject/Level0.tscn

```diff
@@ -13,7 +13,6 @@
 [ext_resource path="res://Spike.tscn" type="PackedScene" id=11]
 [ext_resource path="res://Level0.gd" type="Script" id=12]
 [ext_resource path="res://EOL.tscn" type="PackedScene" id=13]
-[ext_resource path="res://Menus/InGameMenu/InGameMenu.tscn" type="PackedScene" id=14]
```

Comment:
The in-game menu was moved to be inside the Global.tscn scene, so that it doesn't need to be included in each map. I thought it was appropriate to have it there since we also have the fading code there.

Reply:
good call making it global